test/scholar/naive_bayes/gaussian_naive_bayes.exs

```diff
@@ -0,0 +1,267 @@
+defmodule Scholar.NaiveBayes.GaussianTest do
```

Comment:
You are missing the _test.exs extension on this file, so these tests are not running. :)